### DISCUSSION TITLE
How to use directiveComponent composable.

### DISCUSSION THREAD

**Discussion by 4thminor** [vuetifyjs/vuetify#19420] [Q&A] [answered]
I’d like to implement a tooltip directive for multiple buttons within a table and there’s a new [(feature)](https://github.com/vuetifyjs/vuetify/pull/17395) added in version 3.4 that might be the solution. Unfortunately, this feature isn’t documented yet. When I attempted to copy the example codes from that pull request, I encountered a ‘Could not find a declaration file for module’ error.

**Comment by KaelWD** [comment, marked as answer]
We're currently not exposing that, there is a tooltip directive coming soon: https://github.com/vuetifyjs/vuetify/pull/19053

**Reply by 4thminor** [reply]
Thanks, @KaelWD
